wolox-books/src/App.vue

```diff
@@ -1,11 +1,6 @@
-<template>
-  <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
+<template lang="pug">
+  .div(id='app')
```

Comment:
I think this is old, remember to keep the branches up to date and change the base to the previous PR